ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3819,6 +3819,48 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-ipsec
+- as: vsphere-ipi-ovn-dualstack-ipsec-f28
```

Comment:
Based on already have some other `vsphere-ipi-ovn-dualstack` and `vsphere-ipi-ovn-dualstack-privmaryv6`, can we across-cover them to save some vsphere slice quota (we only have a small number of vsphere slice quota).

E.g: non-destructive testing run on `vsphere-ipi-ovn-dualstack`, destructive testing run on `vsphere-ipi-ovn-dualstack-ipsec`.

Reply:
or if we can only do base health check and do not run `openshift-e2e tests on vsphere-ipi-ovn-dualstack`
since `vsphere-ipi-ovn-dualstack-privmaryv6-ipsec-f28` should be able to cover all tests (with or w/o ipsec cases)

Reply:
I'd rather drop IPv4 only tests like `vsphere-ipi-ovn-ipsec-f28`.  The single stack IPv4 tests are less useful.

Reply:
We still have almost a whole matrix test on the Dualstacks.

* ipi-ovn-shared-to-local-gw-migration
* ipi-proxy-fips
* ipi-proxy-workers-rhel8   (we haven't tested DualStack with RHEL workers?)
* ipi-external-lb
* ipi-disk-encryption-tang-fips

All these need testing on the DualStacks.

Ideally DualStack IPv4 would become the default and replace SingleStack IPv4.

Reply:
https://github.com/openshift/release/pull/44812

It looks like we have a bunch of OpenShiftSDN tests as well, those need to be switched to OVN DualStack for 4.15

Reply:
Same for Privmaryv6  https://github.com/openshift/release/pull/44813

Reply:
> I'd rather drop IPv4 only tests like vsphere-ipi-ovn-ipsec-f28

 I am okay with that, beside that, I still have concern about vsphere slice quota, especially after vsphere vpn migration, any install  which need a bastion (e.g: proxy and disconnected mirror) would consume 2 slice quota at least.

if we double (dualstack and dualstack-privmaryv6) all the configurations, slice quota would definitely become a bottleneck for ci testing.

Reply:
@rbbratta I think we can mix all related feature with dualstack to one job. eg

ipi-ovn-dualstack-ipsec-fips
ipi-ovn-dualstack-ipsec-ipv6primary-fips
ipi-disk-encryption-tang-fips  --> for this one, I'm not sure it's related to dualstack, 
for RHEL worker on dualstack I think it's low priority.

Reply:
How do we get more quota?  DualStack is an important feature.

Reply:
I think we have to sent request to SPLAT team.